packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -117,18 +117,58 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  // The ButtonBar widget is planned to be deprecated in favor of the OverflowBar
+  // widget. This ButtonBar implementation uses OverflowBar widget internally
+  // for backward compatibility.
```

Comment:
we should make it clear that the `ButtonBar` widget in `package:flutter/material.dart` is the one that's deprecated, and that the one here is never going to go away.

Reply:
Thanks! Updated the docs.